src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
+* {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
 }
```

Comment:
Consider moving these styles to a separate file to keep your main file clean and clear